src/librustdoc/html/render/mod.rs

```diff
@@ -869,6 +869,7 @@ fn render_assoc_item(
     link: AssocItemLink<'_>,
     parent: ItemType,
     cx: &Context<'_>,
+    render_mode: RenderMode,
```

Comment:
To make the code easier to read, I decided to pass `RenderMode` instead of a boolean to have a continuity with the rest of the code.